.changeset/early-news-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": major
+---
+
+Remove lastUpdatedAt argument from update mutations
```

Comment:
```suggestion
---
"@comet/cms-admin": major
"@comet/cms-api": major
---

CRUD Generator: Remove `lastUpdatedAt` argument from update mutations
```

This is also a change in the Admin Generator, so I'd add both cms-api and cms-admin